src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount = days * 40;
+
+  if (days >= 7) {
+    amount -= 50;
+  } else if (days >= 3) {
+    amount -= 20;
```

Comment:
checklist p 1, it is better to make tree variables for the discount per week and per three days, and price for day, it would be easier to change the code if the price will change